kubeadm/flannel/flannel-overlay.yml

```diff
@@ -63,6 +83,49 @@ data:
         ]
       }
     }
+  cni-conf-containerd.json: |
+    {
+      "name": "flannel.4096",
+      "cniVersion": "0.2.0",
+      "type": "flannel",
+      "capabilities": {
+        "portMappings": true,
+        "dns": true
+      },
+      "delegate": {
+        "type": "sdnoverlay",
+        "AdditionalArgs": [
+          {
+            "Name": "EndpointPolicy",
+            "Value": {
+              "Type": "OutBoundNAT",
+              "Settings" : {
+                "Exceptions": []
+              }
+            }
+          },
+          {
+            "Name": "EndpointPolicy",
+            "Value": {
+              "Type": "SDNROUTE",
+              "Settings": {
+                "DestinationPrefix": "",
+                "NeedEncap": true
+              }
+            }
+          },
+          {
+            "Name":"EndpointPolicy",
+            "Value":{
+              "Type":"ProviderAddress",
+                "Settings":{
+                    "ProviderAddress":""
```

Comment:
@daschott  Does this need to be set to the ip of the host?

https://github.com/microsoft/windows-container-networking/blob/a1b979e8f5cbccd25719747ba20c76ca8d1ae156/example/flannel_overlay.conf#L37-L39

What does it do?

Reply:
I see this is set to the host ip on https://github.com/kubernetes-sigs/sig-windows-tools/pull/112/files/dac8d9cae606b03484fd294b4d2e24a16cd0d9b0#diff-b37151fda8ddf2f86daeb0d2f62f3225e64ce14c0b7f1548cb2ef38bf891e85fR43

Reply:
This gets set in the powershell snippet above (line 43).
I'm not sure what it does but hopefully someone else who can comment on the PR

Reply:
@jsturtevant yes this should be set to the ip of the host. 

I didn't intent for this comment to be blocking initially since I thought this would automatically be added as was the case previously. But upon attempting to remove this, pod-pod connectivity is failing, so this should be included.

Provider Address (PA) is the IP address assigned to the container hosts network infrastructure. The PA appears in the packets on the hosts network. Without the PA, there is no routing for Windows pods and network will not know how get traffic to the right destination host